app/config.py.example

```diff
@@ -27,6 +27,10 @@ PSQL_ENVIRONMENT = False
 MEDIA_DIRECTORY = 'media'
 PICTURE_EXTENSION = '.png'
 AVATAR_SIZE = (120, 120)
+"""For security reasons, set the upload path to a local absolute path.
+Or for testing environment - just specify a folder name
+that will be created under /app/"""
+UPLOAD_DIRECTORY = 'event_images'
```

Comment:
@yammesicka  this is the comment in the config example to explain to users it's preferable to use a local directory.